GOVERNANCE.md

```diff
@@ -88,3 +88,12 @@ As seen above, the different teams under the Node.js Web Team umbrella are respo
 Following this line of thought, the Web Infra Team is responsible for the technical aspects of the Node.js Website (Infrastructure, Framework, CI/CD, etc); The Website Team is responsible for the day-to-day development of the Node.js Website; The UX and Design Team advise on Design Matters and the Web Standards Team advise on best-practices for Web APIs and Web Technologies/Standards.
 
 But above all, the Web Team should work together to better the Web Experience for Node.js, aiming to provide the best experience for Node.js users.
+
+## Blog post publishing guidelines
+
+The following guidelines apply to all blog posts except minor or patch release announcements:
+
+1. each blog post _must_ have a target publishing date and time. If scheduled publishing is not possible, _the author_ (if privileged enough) or another delegated member would be responsible for landing.
+2. each blog post _must_ be approved by @nodejs/tsc members _and_ OpenJS Foundation marketing staff.
```

Comment:
```suggestion
2. the pull requests adding new blog posts should notify @nodejs/tsc and @nodejs/marketing for reviews. In addition, it should request a review from respective teams with corresponding expertise. 

- Release announcements: `@nodejs/releasers`
- Website: `@nodejs/website`
- Security: `@nodejs/security`

If there are no change requests and at least two approvals from the teams being requested reviews from in 48 hours on business days, the blog post is considered approved. If the blog post needs to be fast tracked, a comment should be posted to request fast track, l and at least two thumbs up to the fast track request is needed to approve the blog post.
```

Reply:
I'm strongly -1 on having an automatic approval after any period of time.

Reply:
> If there are no change requests and at least two approvals from the reviewing teams,

I think this is clear enough to indicate that the blog post needs at least two approvals from the reviewing teams?